### ISSUE TITLE
Address measurements that present "unknown failure ascii" based on OONI Probe legacy bug

### ISSUE THREAD

**Issue by agrabeli** [ooni/explorer#481]
The python legacy version of OONI Probe (2.2. series) had a bug that sometimes resulted in the following failure in Web Connectivity measurements:

> unknown_failure 'ascii' codec can't decode byte 0xe2 in position 2886: ordinal not in range(128)

These measurements are flagged as "anomalous" on OONI Explorer, but we actually cannot determine blocking/non-blocking, since this failure renders the measurements useless.

Many people are still running the legacy OONI Probe 2.2. on Linux around the world, continuing to contribute such non-actionable measurements. 

Here is an example of a recent measurement with this failure coming from Belarus: https://explorer.ooni.org/measurement/20200720T024608Z_AS6697_iuQgu6bfYAUtfCGlbaA4cuPF2bA7hZaqyMZ0RBRb65bHaluAmR?input=https%3A%2F%2Fpsiphon.ca%2F

**The "anomalous" indexing of these measurements on OONI Explorer is misleading**, since community members may assume that the tested websites from such measurements are blocked (whereas we cannot determine that when these failures occur). Inspecting the raw measurement data can be challenging, even for technologists.

Given that this is a **known bug**, I propose that we **somehow flag these measurements in such a way that it is clear that they are not actionable, that they should be ignored, and that the user should not infer a testing result based on these measurements.**

**Rather than flagging these measurements as "anomalous", we could flag them as an "error".** We already have measurement pages for errors (which are gray), that we could potentially reuse. In doing so, we can still provide the same measurement data, but provide a gray banner (instead of orange) and say "error" instead of "likely blocked".

We could also potentially add a sort of **notification inside the measurement itself** (similarly to the notification that we previously added for the WhatsApp bug), informing people that this measurement has a failure based on a known bug in the old version of OONI Probe. We could also grab this opportunity to encourage them to stop running the legacy version of OONI Probe, and to install the newer versions instead.

What do you think?

cc' @hellais @sarathms @FedericoCeratto

**Comment by hellais** [comment]
Yes I agree with all that you say. We discussed this in the past with @FedericoCeratto and we believe that the best approach is to do this in the data processing pipeline and eventually expose this in Explorer.

The general direction is that of having less logic in explorer and more logic in the processing pipeline.

IIRC @FedericoCeratto and @bassosimone were collecting all the known unknown failure errors which should be handled in the pipeline.

I believe that once the data processing pipeline is handling these OONI Explorer should "just work" and present these pages as errors.

@sarathms can you confirm if this in fact the case? Like if the pipeline returns `error: true` will you render an error page?

In any case I believe this issue is blocked by: https://github.com/ooni/backend/issues/326.
